lm_eval/tasks/arabic_leaderboard_complete/alghafa/alghafa.yaml

```
@@ -0,0 +1,23 @@
+group: arabic_leaderboard_alghafa
+task:
+  - arabic_alghafa_mcq_exams_test_ar
```

```suggestion
  - arabic_leaderboard_alghafa_mcq_exams_test_ar
```
missing `_leaderboard` for these tasks still I believe?

---

Done